Sources/ResilientDecoding/ResilientPropertyErrorInspection.swift

```diff
@@ -47,21 +47,28 @@ extension Resilient.ProjectedValue where Value: _ResilientSequence {
   /**
    Interleaves encountered errors with the provided set of elements, potentially returning a single error in the case that the top-level decoding has failed.
    */
-  public var results: [Result<Value._Element, Error>] { _errors(interleavedWith: value._resilientElements) }
+  public var results: [Result<Value.Element, Error>] { _errors(interleavedWith: value) }
 
 }
 
-extension Resilient.ProjectedValue where Value: Sequence {
+/**
+ This extension makes it so that Resilient values of type `[T]?` also get array-like `projectedValue` errors using the `@dynamicMemberLookup` annotation in `Resilient.swift`. This was added to avoid conforming `Optional` to a `public` protocol defined in this module.
+ */
+extension Resilient.ProjectedValue {
 
-  /**
-   All errors encountered during decoding
-   */
-  public var errors: [Error] { _errors }
+  public struct _ArrayErrorProperties<T> where T: Sequence {
```

Comment:
Why is this underscored? Might be good to have a comment here that's similar to what you did [here](https://github.com/airbnb/ResilientDecoding/pull/8/files#diff-61d366bc3fc48ac3e874f3a4019a8505L62). We should strive for all of our public types and methods to have documentation.